observability/observability.go

```diff
@@ -31,18 +31,22 @@ import (
 )
 
 var (
-	mReceiverReceivedSpans = stats.Int64("oc.io/receiver/received_spans", "Counts the number of spans received by the receiver", "1")
-	mReceiverDroppedSpans  = stats.Int64("oc.io/receiver/dropped_spans", "Counts the number of spans dropped by the receiver", "1")
-
-	mExporterReceivedSpans = stats.Int64("oc.io/exporter/received_spans", "Counts the number of spans received by the exporter", "1")
-	mExporterDroppedSpans  = stats.Int64("oc.io/exporter/dropped_spans", "Counts the number of spans received by the exporter", "1")
+	mReceiverReceivedSpans      = stats.Int64("otelsvc/receiver/received_spans", "Counts the number of spans received by the receiver", "1")
```

Comment:
Is the purpose of renaming from oc.io to otelsvc just rebranding?

Reply:
@pjanotti same question as in the other PR: does this require any updates in our internal dashboards?

Reply:
Yes, mostly rebranding. Let's talk about the other question int the other PR.